service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -715,4 +817,18 @@ void shouldSendNotificationToPartiesWhenAddedToCaseThroughDigitalService() throw
             .put("caseUrl", "null/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345")
             .build();
     }
+
+    private CallbackRequest appendEnableCtscOnCallback(CallbackRequest callbackRequest) {
```

Comment:
you could just declare callbackRequest() in here, no need to pass it in is there?

Reply:
Yeah good point 👍